sources/ConflictWindow.cpp

```diff
@@ -11,31 +11,39 @@
 #include "ConflictWindow.h"
 
 #include <Button.h>
+#include <Catalog.h>
 #include <ControlLook.h>
 #include <LayoutBuilder.h>
 #include <String.h>
 #include <StringView.h>
 
+#undef B_TRANSLATION_CONTEXT
+#define B_TRANSLATION_CONTEXT "ConflictWindow"
+
 static const unsigned kReplace = 'RPLC';
 static const unsigned kSkip = 'SKIP';
 
 
 ConflictWindow::ConflictWindow(const char* file)
 	:
-	BWindow(BRect(0, 0, 0, 0), "Filer: Conflict", B_FLOATING_WINDOW_LOOK, B_FLOATING_ALL_WINDOW_FEEL,
-		B_NOT_CLOSABLE | B_NOT_ZOOMABLE | B_NOT_RESIZABLE | B_AVOID_FOCUS | B_AUTO_UPDATE_SIZE_LIMITS),
-	fDoAll(new BCheckBox("", "Do this for all files", NULL)),
+	BWindow(BRect(0, 0, 0, 0), B_TRANSLATE("Filer: Conflict"),
+		B_FLOATING_WINDOW_LOOK, B_FLOATING_ALL_WINDOW_FEEL,
+		B_NOT_CLOSABLE | B_NOT_ZOOMABLE | B_NOT_RESIZABLE | B_AVOID_FOCUS |
+		B_AUTO_UPDATE_SIZE_LIMITS),
+	fDoAll(new BCheckBox("", B_TRANSLATE("Do this for all files"), NULL)),
 	fReplace(false),
 	fSem(create_sem(0, ""))
 {
 	const float spacing = be_control_look->DefaultItemSpacing();
 
-	BString text(file);
-	text += " already exists.";
+	BString text(B_TRANSLATE_COMMENT("%file% already exists.",
```

Comment:
I suggest to place the %file% variable in single quotes: '%file%' already exists...
If the filename is "Invoice_0887-17.docx" it's obvious. If the filename is "Something", things are less so... :)